apps/wallet/src/ui/app/pages/accounts/manage/AccountGroup.tsx

```diff
@@ -134,9 +134,9 @@ export function AccountGroup({
 				<VerifyPasswordModal
 					open
 					onVerify={async (password) => {
-						if (accountsFormValues && accountsFormValues.type !== 'zk') {
+						if (accountsFormValues.current && accountsFormValues.current.type !== 'zk') {
```

Comment:
Using refs here does scare me a bit because we don't trigger re-renders on value changes. I think it's fine to make this land now, but it's a solid indicator we've done something wrong architecturally.

Reply:
Yeah there are some places where we call a mutation right after setting the values in the same callback and the mutation uses the previous values.
We don't really do any rendering of these just use them for keeping state between views and on user action.

Reply:
Yeah I think for the sake of getting this out it's fine, it's just still an indicator that we're doing something wrong somewhere.